test/DynamoCoreTests/PythonMigrationAssistantTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Dynamo.PythonMigration.MigrationAssistant;
+using NUnit.Framework;
+using System.Collections.Generic;
+
+namespace Dynamo.Tests
+{
+    public class PythonMigrationAssistantTests
+    {
+        [Test]
+        public void CanMigratePyton2CodeToPython3Code()
+        {
+            // Arrange
+            var expectedPython3Code = @"print(""Hello, Dynamo!"")";
+
+            var python2Code = @"print ""Hello, Dynamo!""";
+
+            var inputNames = new List<string>()
+            {
+                "code"
+            };
+            var inputValues = new List<object>()
```

Comment:
would we ever have more than 1 value we pass in ? if no, it's another implementation detail that should not be visible from the outside, it'd be a much cleaner API & more expected to call `ScriptMigrator.MigrateCode(code, "output");

Not sure about the `output` being required either.

Reply:
yea true. I changed this so that `code` and `output` are both `const` on the `ScriptMigrator` - So it is now call like this ScriptMigrator.MigrateCode(code)`